.github/dependabot.yaml

```diff
@@ -76,6 +80,8 @@ updates:
     # Ignore wrangler
     - dependency-name: "github.com/rancher/wrangler"
     - dependency-name: "github.com/rancher/wrangler/v2"
+    # Ignore rancher apis
+    - dependency-name: "github.com/rancher/rancher/pkg/apis"
```

Comment:
Should we add this to `release-v2.8` as well?

Reply:
added, thank you